### DISCUSSION TITLE
Setup Manager and HTTPS Authentication

### DISCUSSION THREAD

**Discussion by peteostro** [jamf/Setup-Manager#65] [General]
We hare looking at setup manager to replace our Depnotify config as its not supported any more. We have Jamf Pro on Prem

We have setup an HTTPS distribution point on an IIS server with anonymous share access added the signed setup manager pkg, uploaded the manifest file and setup the prestage enrollment package to deploy setup manager.

Setup manager launches and starts configuring and trying to deploy packages

When trying to deploy packages (from another HTTPS distribution point with authentication required) the deployments fail. Is this the case that all packages you want to deploy with setup manager need to be setup on an anonymous share accessible to all? meaning licensed software can be downloaded by anyone that knows the URL (on campus at least). Maybe I am missing something but this seems like a bad idea?
When I do enable anonymous share access and set authentication type to none on the distribution share the packages deploy.

**Comment by scriptingosx** [comment]
@Gregzy1987 provided this during the private beta phase

> When deploying this using an SMB share, Setup Manager installs fine but any policy called by Setup Manager fails to install and appears it needs a PPPC to allow com.jamf.setupmanager access to SystemPolicyAllFiles.

**Reply by peteostro** [reply]
Adding the PPPC to prestage profile (giving setup manager SystemPolicyAllFiles allow) worked and the smb distribution share was able to be mounted and packages deployed. Would like to know why HTTPS deployment wouldn’t work though.
